src/components/Actions.tsx

```diff
@@ -114,6 +118,14 @@ export const SelectedShapeActions = ({
         <>{renderAction("changeSharpness")}</>
       )}
 
+      {!appState.draggingElement &&
+        nonTextElements.length > 0 &&
+        nonTextElements.length < 2 &&
```

Comment:
to help deciding whether to keep the slider, let's show it when selecting multiple elements (or did you disable for some specific reason?).